nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,92 @@
+const express = require("express");
+const app = express();
+
+
+// import data here
+const meals = require("./data/meals.json");
+const reservations = require("./data/reservations.json");
+const reviews = require("./data/reviews");
+//console.log(meals[0].title);
+
+// this is where you will be adding your routes
```

Comment:
Bonus:
One common pattern that you will see in the future is to extract the route handler to a named function (and sometimes that function is put into a router or controller object). 

So you would end up with code like this:
```
// All route definitions go in one block
app.get("/", sendGreeting);
app.get("/meals", listMeals);
app.get("/reservations", listReservations);

// One block/function per route handler
async function sendGreeting(request, response) {
 ...
}

async function listMeals(request, response) {
 ...
}
async function listReservations(request, response) {
 ...
}
```

The benefit of this style is that the next person reading this code can quickly tell which routes exist and the name of the handler can give more information about what this route returns. Then if they want to know the details, they can read the code in only the handler they are looking for.